src/Engine/App/TransformController.cpp

```
@@ -27,14 +27,13 @@ CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
 #include "App/MouseEventListener.h"
 
-namespace OSRE {
-namespace App {
+namespace OSRE::App {
 
 using namespace ::OSRE::Animation;
 using namespace ::OSRE::RenderBackend;
 using namespace ::OSRE::Platform;
 
-static constexpr c8 Tag[] = "TransformController";
+DECL_OSRE_LOG_MODULE(TransformControlle)
```

Cppcheck (2.10-2)

[error] There is an unknown macro here somewhere. Configuration is required. If DECL_OSRE_LOG_MODULE is a macro then please configure it.

(unknownMacro)